scripts/pugme.js

```diff
@@ -0,0 +1,28 @@
+// Description:
+//   Pugme is the most important thing in life
+//
+// Dependencies:
+//   None
+//
+// Configuration:
+//   None
+//
+// Commands:
+//   randbot pug me - Receive a pug
+//   randbot pug bomb N - get N pugs (max 10)
+
+module.exports = function (robot) {
+  robot.respond(/pug me/i, (msg) => {
+    msg.http('http://pugme.herokuapp.com/random').get()((err, res, body) => {
+      msg.send(JSON.parse(body).pug);
+    });
+  });
+  robot.respond(/pug bomb( (\d+))?/i, (msg) => {
```

Comment:
I'm going to have to dig into the second capture group thing a little then. I don't quite understand why it's necessary but I'm not going to hold up this code for it.